src/main/java/org/broadinstitute/hellbender/engine/VariantWalker.java

```diff
@@ -63,13 +69,27 @@ protected final void onStartup() {
         }
     }
 
+    @Override
+    void initializeFeatures() {
+        //Note: we override this method because we don't want to set feature manager to null if there are no FeatureInputs.
+        //This is because we have at least 1 source of features (namely the driving dataset).
+        features = new FeatureManager(this);
```

Comment:
Can we move the code that initializes the driving variants source into a private `initializeDrivingVariants()` method, and call it from here (`initializeFeatures()`) rather than from `onStartup()`?